benchmarks/hf.sh

```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
 source utils.sh
 if [[ $SKIP_BASELINE -eq 0 ]]; then
-    export BASELINE_REPO=$CACHE_DIR/transformers_v3.0.2
-    #https://github.com/huggingface/transformers.git \
+    export BASELINE_REPO=$CACHE_DIR/transformers_v4.12.0
     git_clone_if_not_in_cache \
-	https://github.com/JiushengChen/transformers.git \
```

Comment:
Some context about this forked repo, it is to add param "no_repeat_ngram_size", see
https://github.com/JiushengChen/Transformers/commit/db970431a6d55d5b1477e21ff719612e425c5f95

Reply:
To clarify, this is no longer needed since fastseq uses it's own run_eval_hf.py for the baseline?